clusterman/migration/event.py

```diff
@@ -131,7 +133,7 @@ class MigrationEvent(NamedTuple):
 
     def __hash__(self) -> int:
         """Simplified object hash since resource_name should be unique"""
-        return self[:3].__hash__()
+        return cast(Tuple[str, str, str], self[:3]).__hash__()
```

Comment:
huh, interesting that mypy couldn't figure this out

Reply:
It was complaining with 

```
error: Invalid self argument "MigrationEvent" to attribute function "__hash__" with type "Callable[[MigrationEvent], int]"
```

It basically cannot figure out that `MigrationEvent` is effectively a tuple, and a slice of itself is no longer the same type.